.github/workflows/pr-manual-merge.js

```diff
@@ -0,0 +1,79 @@
+const { slackNotification, getLocalConfigs } = require('./helpers.js');
+
+async function main({ github, context } = {}) {
+    if (!github || !context) {
+        throw new Error("GitHub context is missing. Ensure you are running in the correct environment.");
+    }
+
+    if (process.env.LOCAL_RUN) {
+        console.log("Local run detected. Loading local configurations...");
+        const localConfigs = getLocalConfigs();
+        github = localConfigs.github;
+        context = localConfigs.context;
+    }
+
+    const { pull_request } = context.payload;
+    if (!pull_request) {
+        console.log("No pull_request found in context payload. Skipping notification.");
+        return;
+    }
+
+    const { number, title, html_url, base } = pull_request;
+    const isStage = base.ref === 'stage';
+    const prefix = isStage
+        ? ':merged: PR merged to stage:'
+        : ':rocket: Production release:';
+
+    console.log(`Sending notification for PR #${number}: ${title}`);
+
+    try {
+        await slackNotification(
+            `${prefix} <${html_url}|#${number}: ${title}>.`,
+            process.env.MILO_RELEASE_SLACK_WH
+        );
+    } catch (error) {
+        console.error("Error sending Slack notification:", error.message);
+        console.log("Continuing with PR update...");
+    }
+    
+    if (['stage', 'main'].includes(base.ref)) {
```

Comment:
We don't want to update the description for the stage PR, if we merged something into main. It's not "part" of a batch, but already on prod.